docs/examples/QuTech_M2b.ipynb

```
@@ -0,0 +1,286 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f728c618",
+   "metadata": {},
+   "source": [
+    "## Virtual Driver"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Ruff (0.14.3)

Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)